clang/lib/CConv/CheckedRegions.cpp

```diff
@@ -137,13 +153,6 @@ bool CheckedRegionFinder::hasUncheckedParameters(CompoundStmt *S) {
 }
 
 
-bool CheckedRegionFinder::VisitUnaryOperator(UnaryOperator *U) {
-  //TODO handle computing pointers
-  if (U->getOpcode() == UO_AddrOf) {
-    // wild++;
-  }
-  return true;
-}
```

Comment:
This was just irrelevant as far as `_Checked` regions goes, I imagine?

Reply:
Yep. Thought I needed it at one point, ended up not needing it but it stayed in the code